examples/tutorial/hello/go/README.md

```diff
@@ -1,66 +1,138 @@
-# Tutorial 1: Go Function w/ Input (3 minutes)
+# Go Function Hello World
 
-This example will show you how to test and deploy Go (Golang) code to Fn. It will also demonstrate passing data in through stdin.
+This example will shows you how to test and deploy Go code to Fn. It will also demonstrates passing JSON data to your function through `stdin`.
 
-### First, run the following commands:
+This tutorial assumes you have installed Docker, Fn server, and Fn CLI. See the [Fn Quickstart](https://github.com/fnproject/fn) for installation steps.
+
+## Start Fn Server
```

Comment:
Maybe just link this to another doc?  If we repeat this everywhere, it will surely get out of sync. Could be part of the assumptions above.